src/highlight/HighlightTarget.tsx

```diff
@@ -14,13 +14,13 @@ type Props = {
     rects: Array<Rect>;
 };
 
-export type HighlightAnnotationRef = HTMLAnchorElement;
+export type HighlightTargetRef = HTMLAnchorElement;
 
-const HighlightAnnotation = (props: Props, ref: React.Ref<HighlightAnnotationRef>): JSX.Element => {
+const HighlightTarget = (props: Props, ref: React.Ref<HighlightTargetRef>): JSX.Element => {
```

Comment:
Is this change necessary? Seems like it may be simpler to leave it as-is for consistency with regions?

Reply:
Yeah, I could see the benefit of that -- I also liked your suggestion of making it `HighlightTarget` since it's only the interact-able piece of the highlight. `HighlightAnnotation` feels like it could be misleading since the canvas is the visible portion of the highlight